resources/wikia/ui_components/modal/css/modal_default.scss

```diff
@@ -79,6 +79,11 @@ $large-modal-margin: 50px;
 		width: 20px;
 	}
 
+  	> header .close-counter {
```

Comment:
I'm not sure if we should have it here or since it's ads specific move it to our `ads.scss`. Will it work? What are your thoughts?